content/posts/newsletter-007/index.md

```diff
@@ -76,6 +76,28 @@ Features:
 [rusty-shooter]: https://github.com/mrDIMAS/rusty-shooter
 [rusty-shooter-video]: https://www.youtube.com/watch?v=UDn8ymyXPcI
 
+### [Lonely Star][lonely-star]
+
+> *Space is a lonely place, but at least you've got the music to keep you company.*
+
+[Lonely Star][lonely-star] is a side-scrolling infinite runner, with simple generative music. You collect orbs and fly through rings in order to play notes and stay alive.
+
+It was developed by [17cupsofcoffee][17cupsofcoffee], using the [Tetra][tetra] 2D game framework, for [Weekly Game Jam #135](weekly-game-jam-135).
+
+[![Lonely Star screenshot](lonely-star.PNG)][lonely-star]
+A
```

Comment:
Also, this lonely A looks a little bit weird)

Reply:
Agh, I think I mashed my keyboard by mistake when committing 😆 Will fix.

Reply:
Fixed, and rebased on top of the other PR that just merged,